Dockerfile

```diff
@@ -20,7 +20,11 @@ WORKDIR /stage/allennlp
 
 # Install the wheel of AllenNLP.
 COPY dist dist/
-RUN pip install $(ls dist/*.whl)
+RUN pip install $(ls dist/*.whl)[vision]
+
+# Install detectron2.
+RUN pip install detectron2 -f https://dl.fbaipublicfiles.com/detectron2/wheels/cu102/torch1.7/index.html
```

Comment:
This line has the CUDA version and the torch version built in. Is there a way to not be dependent on that?

Reply:
Yea, good point. We could (and should) make that a build variable.

Reply:
Once https://github.com/allenai/allennlp/pull/4796 goes through I'll add a build ARG for detectron.